packages/node/src/integrations/contextlines.ts

```diff
@@ -0,0 +1,141 @@
+import { getCurrentHub } from '@sentry/core';
+import { Event, EventProcessor, Integration } from '@sentry/types';
+import { addContextToFrame } from '@sentry/utils';
+import { readFile } from 'fs';
+import { LRUMap } from 'lru_map';
+
+import { NodeClient } from '../client';
+
+const FILE_CONTENT_CACHE = new LRUMap<string, string | null>(100);
+const DEFAULT_LINES_OF_CONTEXT = 7;
+
+function readTextFileAsync(path: string): Promise<string> {
+  return new Promise((resolve, reject) => {
+    readFile(path, 'utf8', (err, data) => {
+      if (err) reject(err);
+      else resolve(data);
+    });
+  });
+}
```

Comment:
```suggestion
import { promisify } from 'util';

const readTextFileAsync = promisify(readFile);
```

Docs: https://nodejs.org/api/util.html#utilpromisifyoriginal

Reply:
I'm guessing this came from before node v8 was the minimum 🤔

Reply:
Ah, the issue with `promisify` is that it doesn't let you pass `utf8` which means it will return a `Buffer` rather than `string`.

Reply:
```ts
const readFileAsync = promisify(readFile);

// I assume this is all node does anyway to return a string?
const textFile = (await readFileAsync(path)).toString();
```

Reply:
hehe happens to the best of us!